database/src/main/scala/cromwell/database/obj/WorkflowMetadataSummary.scala

```diff
@@ -42,9 +39,9 @@ case class WorkflowMetadataSummary
     // Resolve the status if both `this` and `that` have defined statuses.  This will evaluate to `None`
     // if one or both of the statuses is not defined.
     val resolvedStatus = for {
-      a <- this.status map WorkflowState.fromString
-      b <- that.status map WorkflowState.fromString
-    } yield (a |+| b).toString
+      thisStatus <- this.status
+      thatStatus <- that.status
+    } yield (thisStatus |+| thatStatus).toString
```

Comment:
I don't think you need the `.toString` or the parens around your fancy `Semigroup` append 😛